Extensions/EdFi.Ods.Extensions.SampleStudentTransportation/EdFi.Ods.Extensions.SampleStudentTransportation.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0`

Reply:
updated